src/5e-SRD-Spells.json

```diff
@@ -130,13 +130,13 @@
     "higher_level": [
       "When you cast this spell using a spell slot of 3rd level or higher, a target's hit points increase by an additional 5 for each slot level above 2nd."
     ],
-    "range": "30 feet",
+    "range": {"value": 30, "unit": "ft"},
     "components": ["V", "S", "M"],
     "material": "A tiny strip of white cloth.",
     "ritual": false,
-    "duration": "8 hours",
+    "duration": {"value": 8, "unit": "hrs"},
```

Comment:
All the other units are in the singular, shouldn't `hrs` be `hr` or `hour`?

Reply:
Searched on the web how hours is usually abbreviated and `hrs` came up. Maybe just `hour` should work, will replace.